src/lib.rs

```diff
@@ -324,6 +324,43 @@ pub struct WalletOpts {
     #[cfg(feature = "esplora")]
     #[structopt(flatten)]
     pub esplora_opts: EsploraOpts,
+    #[cfg(feature = "compact_filters")]
+    #[structopt(flatten)]
+    pub compactfilter_opts: CompactFilterOpts,
+}
+
+/// Compact Filter options
+///
+/// Compact filter peer information used by [`OnlineWalletSubCommand`]s.
+#[cfg(feature = "compact_filters")]
+#[derive(Debug, StructOpt, Clone, PartialEq)]
+pub struct CompactFilterOpts {
+    /// Sets the SOCKS5 proxy for the Electrum client
+    #[structopt(name = "CPROXY_SERVER:PORT", long = "cmpct_proxy")]
+    pub proxy: Option<String>,
+
+    #[structopt(name = "PROXY_SERVER:USER", short = "u", long = "user")]
+    pub user: Option<String>,
+
+    #[structopt(name = "PROXY_SERVER:PASSWD", long = "passwd")]
+    pub passwd: Option<String>,
+
+    /// Sets the light client network address
+    #[structopt(name = "ADDRESS:PORT", short = "n", long = "node")]
```

Comment:
Similar to ElectrumOpts server, I'd set a `default_value` for CompactFiltersOpts address ("node") to "127.0.0.1:18444", hopefully in the future we'll be able to make this optional and find our peers via dns seed discovery if none are given. But for now I think it's ok to just default to what most people will probably be testing with. This also give an example to show users the expected format.  I'd also add to the description to make it clear this is peer full node address, something like: "Sets the compact block filter enabled full node address"

Reply:
Yes agreed. Also enabling `conn_coun` to start multiple connections to the same node. This parameter is not used right now. 

Also thinking about having `--node` as a list of `ADDRS:PORT` strings. Instead of one address, user can give multiple addresses and wallet will make a total of `no. of address * conn_coun` connections.

Does that make sense?

Reply:
Sounds good but I don't think anyone has tested the CBF blockchain code with multiple nodes, just with multiple connections to the same node. But seems like it should work and maybe now is a good time to test it. Worst case we go back to just connecting to a single node multiple times.

Reply:
Ya. Updated the code. Planning to run that exact test too. Even if a multiple node connection doesn't work, the user can still give a single node address and it will resume standard behavior.